src/mix_all.c

```diff
@@ -203,6 +203,10 @@
 
 #endif
 
+#ifdef _MSC_VER
```

Comment:
This seems like a very dirty fix to me. Variable shadowing can cause real bugs and should not be ignored.

Reply:
Those particular ones in mix_all.c are intentional: https://github.com/libxmp/libxmp/pull/402#issuecomment-871321561

I don't want to touch the ones in vorbis.c, because it's an external source.

Reply:
The mixer macros really are written in a way where they rely on shadowing here. I think the filter macros would redefine the volume vars so another set of macros would use the filter output instead, or something similarly silly.

It's something I can look at when I'm adding stereo samples.